### DISCUSSION TITLE
开源的不可避免性 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#84] [Announcements]
# 开源的不可避免性 - 「开源之道」

https://opensourceway.community/posts/open-source-frontier/what-are-open-source-precondation/

**Comment by ZoomQuiet** [comment]
#是也乎
作为一篇活动广告, 是及格的, 
可惜没有构建出合格的 MEME;

比如: 开源的前沿是社会;
可其实, 社会的前沿才是开源,
只有在开源中, 有各种最前沿的社会行为,
比如, BTC , 是当前, 也可能是人类历史上最前沿的社会革命了;

开源不可避免是建立在开源是免费的基础上的;
如果有金主要求不准使用任何一行开源代码, 
也是可以轻易作到的,
嘦拿开源项目来, 逐行改写...

所以, 不可避免, 不是技术层面上的, 而是社会道义上的,
对重复劳动从灵魂中的反抗...
